<ui version="4.0" >
 <class>SignalSlotDialogClass</class>
 <widget class="QDialog" name="SignalSlotDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Signals and slots</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="slotGroupBox" >
     <property name="title" >
      <string>Slots</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QListView" name="slotListView" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QToolButton" name="addSlotButton" >
          <property name="toolTip" >
           <string>Add</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeSlotButton" >
          <property name="toolTip" >
           <string>Delete</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="signalGroupBox" >
     <property name="title" >
      <string>Signals</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QListView" name="signalListView" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QToolButton" name="addSignalButton" >
          <property name="toolTip" >
           <string>Add</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeSignalButton" >
          <property name="toolTip" >
           <string>Delete</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="spacerName" stdset="0" >
           <string/>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
